pages/mods.vue

```diff
@@ -583,7 +652,6 @@ export default {
           window.history.replaceState(new Date(), 'Mods', url)
         }
       } catch (err) {
-        // eslint-disable-next-line no-console
```

Comment:
Keep the eslint checker tho, as it could cause issues when having eslint enabled (for example in the CI runs).

Reply:
Sorry, I thought that I added those when I added my console.log()s. I added them back in 3d9be6d.